api/accounts/models.py

```diff
@@ -23,7 +23,7 @@ class Profile(models.Model):
         on_delete=models.CASCADE, primary_key=True
     )
     topics = models.ManyToManyField('community.Topic')
-    image = models.ImageField(default='default.jpg', upload_to='profile_pics')
+    image = models.CharField(max_length=150, blank=True)
```

Comment:
Where did "max_length=150" come from? What happens if the user uploads a file with (for example) a 255-letter filename?

Reply:
@Underwaterr the file name is set by a folder name and a UUID :)